<!DOCTYPE html>

<html lang="en" data-content_root="./">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>Alignmenttrees &#8212; WGDI 0.71 documentation</title>
    <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=d1102ebc" />
    <link rel="stylesheet" type="text/css" href="_static/alabaster.css?v=12dfc556" />
    <script src="_static/documentation_options.js?v=89bdf08e"></script>
    <script src="_static/doctools.js?v=9bcbadda"></script>
    <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Retain" href="retain.html" />
    <link rel="prev" title="Alignment" href="alignment.html" />
   
  <link rel="stylesheet" href="_static/custom.css" type="text/css" />
  

  
  

  </head><body>
  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          

          <div class="body" role="main">
            
  <section id="alignmenttrees">
<h1>Alignmenttrees<a class="headerlink" href="#alignmenttrees" title="Link to this heading">Â¶</a></h1>
<p>Phylogenetic tree construction using alignment or collinear genes.
A phylogenetic tree is constructed for gene sets that each row of the alignment file during this process. By using ASTRAL merge, you can obtain a multispecies or subgenomes coalescent tree.</p>
<p class="rubric">Parameters</p>
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Parameters</p></td>
<td><p>Standards and instructions</p></td>
</tr>
<tr class="row-even"><td><p>alignment</p></td>
<td><p>Type: <strong>file</strong>  |      Default: <strong>-</strong></p>
<p>The merged file of multiple alignment files with the same reference.</p>
</td>
</tr>
<tr class="row-odd"><td><p>gff</p></td>
<td><p>Type: <strong>file</strong>  |      Default: <strong>-</strong></p>
<p>Gff of reference, If alignment has no reference species, delete it.</p>
</td>
</tr>
<tr class="row-even"><td><p>lens</p></td>
<td><p>Type: <strong>file</strong>  |    Default: <strong>-</strong></p>
<p>lens of reference, If alignment has no reference species, delete it.</p>
</td>
</tr>
<tr class="row-odd"><td><p>dir</p></td>
<td><p>Type: <strong>folder</strong> |  Default: <strong>-</strong></p>
<p>Folder for phylogenetic trees.</p>
</td>
</tr>
<tr class="row-even"><td><p>sequence_file</p></td>
<td><p>Type: <strong>file</strong>  |    Default: <strong>-</strong></p>
<p>In general, it is protein sequences (pep), if it is coding sequences (cds), cds_file need to be deleted.</p>
</td>
</tr>
<tr class="row-odd"><td><p>cds_file</p></td>
<td><p>Type: <strong>file</strong>   | Default: <strong>none</strong></p>
<p>It is required when the method of constructing trees involvesv codon. Otherwise, it can be discarded.</p>
</td>
</tr>
<tr class="row-even"><td><p>codon_positon</p></td>
<td><p>Type:<strong>str</strong>  |      Default: <strong>1,2,3</strong></p>
<p>1,2 mean codon1&amp;2; 1,2,3 mean no codon removed.</p>
</td>
</tr>
<tr class="row-odd"><td><p>trees_file</p></td>
<td><p>Type: <strong>file</strong>  |    Default: <strong>-</strong></p>
<p>Merge multiple nwk-format tree files.</p>
</td>
</tr>
<tr class="row-even"><td><p>align_software</p></td>
<td><p>Type:{ <strong>muscle</strong>, <strong>mafft</strong> }  |   Default: <strong>muscle</strong></p>
<p>Software of multiple sequence alignment.</p>
</td>
</tr>
<tr class="row-odd"><td><p>tree_software</p></td>
<td><p>Type:{ <strong>iqtree</strong>, <strong>fasttree</strong> }  |   Default: <strong>iqtree</strong></p>
<p>Software of constructing phylogenetic trees.</p>
</td>
</tr>
<tr class="row-even"><td><p>model</p></td>
<td><p>Type: <strong>str</strong>  |    Default: <strong>-</strong></p>
<p>Merge multiple nwk-format tree files.</p>
</td>
</tr>
<tr class="row-odd"><td><p>trimming</p></td>
<td><p>Type:{ <strong>trimal</strong>, <strong>divvier</strong>}  |   Default: <strong>trimal</strong></p>
<p>Software of removal of spurious sequences.</p>
</td>
</tr>
<tr class="row-even"><td><p>minimum</p></td>
<td><p>Type: <strong>int</strong>  |   Default: <strong>4</strong></p>
<p>Minimum number of gene sets in constructing phylogenetic tree.</p>
</td>
</tr>
<tr class="row-odd"><td><p>delete_detail</p></td>
<td><p>Type: <strong>bool</strong>   |    Default: <strong>false</strong></p>
<p>Whether or not to keep intermediate files when constructing phylogenetic tree.</p>
</td>
</tr>
</tbody>
</table>
<p>Use command to enter the folder <code class="docutils literal notranslate"><span class="pre">wgdi</span> <span class="pre">-at</span> <span class="pre">?</span> <span class="pre">&gt;&gt;</span> <span class="pre">total.conf</span></code> Take out the parameter file.
Afterwards, use the ASTRAL command <code class="docutils literal notranslate"><span class="pre">java</span> <span class="pre">-jar</span> <span class="pre">/path/astral.5.7.7.jar</span> <span class="pre">-i</span> <span class="pre">trees_file.nwk</span> <span class="pre">-o</span> <span class="pre">out.tre</span> <span class="pre">-t</span> <span class="pre">8</span></code>, you can obtain a multispecies or subgenomes coalescent tree.</p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="p">[</span><span class="n">alignmenttrees</span><span class="p">]</span>
<span class="n">alignment</span> <span class="o">=</span> <span class="n">alignment</span> <span class="n">file</span> <span class="p">(</span><span class="o">.</span><span class="n">csv</span><span class="p">)</span>
<span class="n">gff</span> <span class="o">=</span> <span class="n">gff</span> <span class="n">file</span> <span class="p">(</span><span class="n">reference</span> <span class="n">genome</span><span class="p">,</span> <span class="n">If</span> <span class="n">alignment</span> <span class="n">has</span> <span class="n">no</span> <span class="n">reference</span> <span class="n">species</span><span class="p">,</span> <span class="n">delete</span> <span class="n">it</span><span class="p">)</span>
<span class="n">lens</span> <span class="o">=</span> <span class="n">lens</span> <span class="n">file</span> <span class="p">(</span><span class="n">If</span> <span class="n">alignment</span> <span class="n">has</span> <span class="n">no</span> <span class="n">reference</span> <span class="n">species</span><span class="p">,</span> <span class="n">delete</span> <span class="n">it</span><span class="p">)</span>
<span class="nb">dir</span> <span class="o">=</span> <span class="n">output</span> <span class="n">folder</span>
<span class="n">sequence_file</span> <span class="o">=</span> <span class="n">sequence</span> <span class="n">file</span> <span class="p">(</span><span class="o">.</span><span class="n">fa</span><span class="p">)</span>
<span class="n">cds_file</span> <span class="o">=</span> <span class="n">cds</span> <span class="n">file</span> <span class="p">(</span><span class="o">.</span><span class="n">fa</span><span class="p">)</span>
<span class="n">codon_positon</span> <span class="o">=</span> <span class="mi">1</span><span class="p">,</span><span class="mi">2</span><span class="p">,</span><span class="mi">3</span>  <span class="p">(</span><span class="mi">1</span><span class="p">,</span><span class="mi">2</span> <span class="n">mean</span> <span class="n">codon1</span><span class="o">&amp;</span><span class="mi">2</span><span class="p">;</span> <span class="mi">1</span><span class="p">,</span><span class="mi">2</span><span class="p">,</span><span class="mi">3</span> <span class="n">mean</span> <span class="n">no</span> <span class="n">codon</span> <span class="n">removed</span><span class="p">)</span>
<span class="n">trees_file</span> <span class="o">=</span>  <span class="n">trees</span> <span class="p">(</span><span class="o">.</span><span class="n">nwk</span><span class="p">)</span>
<span class="n">align_software</span> <span class="o">=</span> <span class="n">mafft</span>
<span class="n">tree_software</span> <span class="o">=</span>  <span class="p">(</span><span class="n">iqtree</span><span class="p">,</span><span class="n">fasttree</span><span class="p">)</span>
<span class="n">model</span> <span class="o">=</span> <span class="n">MFP</span>
<span class="n">trimming</span> <span class="o">=</span>  <span class="n">trimal</span>
<span class="n">minimum</span> <span class="o">=</span> <span class="mi">4</span>
<span class="n">delete_detail</span> <span class="o">=</span> <span class="n">true</span>
</pre></div>
</div>
<p class="rubric">Quick start</p>
<p>After the parameters are modified properly, then run <code class="docutils literal notranslate"><span class="pre">wgdi</span> <span class="pre">-at</span> <span class="pre">total.conf</span></code></p>
<p class="rubric">Example</p>
<p>The original results are easily accessible at <a class="reference external" href="https://github.com/SunPengChuan/wgdi-example">wgdi-example</a></p>
</section>


          </div>
          
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="Main">
        <div class="sphinxsidebarwrapper">
<h1 class="logo"><a href="index.html">WGDI</a></h1>








<h3>Navigation</h3>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="Introduction.html">Introduction</a></li>
<li class="toctree-l1"><a class="reference internal" href="Installation.html">Installation</a></li>
<li class="toctree-l1 current"><a class="reference internal" href="usage.html">Usage</a><ul class="current">
<li class="toctree-l2"><a class="reference internal" href="dotplot.html">Dotplot</a></li>
<li class="toctree-l2"><a class="reference internal" href="collinearity.html">Improved collinearity</a></li>
<li class="toctree-l2"><a class="reference internal" href="ks.html">Non-synonymous (Ka) and synonymous (Ks)</a></li>
<li class="toctree-l2"><a class="reference internal" href="blockinfo.html">BlockInfo</a></li>
<li class="toctree-l2"><a class="reference internal" href="correspondence.html">Correspondence</a></li>
<li class="toctree-l2"><a class="reference internal" href="blockks.html">BlockKs</a></li>
<li class="toctree-l2"><a class="reference internal" href="kspeaks.html">KsPeaks</a></li>
<li class="toctree-l2"><a class="reference internal" href="peaksfit.html">PeaksFit</a></li>
<li class="toctree-l2"><a class="reference internal" href="ksfigure.html">KsFigure</a></li>
<li class="toctree-l2"><a class="reference internal" href="polyploidy_classification.html">Polyploid classification</a></li>
<li class="toctree-l2"><a class="reference internal" href="ancestral_karyotype.html">ancestral_karyotype</a></li>
<li class="toctree-l2"><a class="reference internal" href="ancestral_karyotype_repertoire.html">ancestral_karyotype_repertoire</a></li>
<li class="toctree-l2"><a class="reference internal" href="karyotype_mapping.html">karyotype_mapping</a></li>
<li class="toctree-l2"><a class="reference internal" href="karyotype.html">Karyotype</a></li>
<li class="toctree-l2"><a class="reference internal" href="shared_fusions.html">Shared_fusions</a></li>
<li class="toctree-l2"><a class="reference internal" href="fusion_positions_database.html">Fusion_positions_database</a></li>
<li class="toctree-l2"><a class="reference internal" href="fusion_detetion.html">Fusions_detection</a></li>
<li class="toctree-l2"><a class="reference internal" href="alignment.html">Alignment</a></li>
<li class="toctree-l2 current"><a class="current reference internal" href="#">Alignmenttrees</a></li>
<li class="toctree-l2"><a class="reference internal" href="retain.html">Retain</a></li>
<li class="toctree-l2"><a class="reference internal" href="pindex.html">P-index</a></li>
<li class="toctree-l2"><a class="reference internal" href="circos.html">Circos</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="Examples.html">Examples</a></li>
<li class="toctree-l1"><a class="reference internal" href="Changelog.html">Changelog</a></li>
</ul>

<div class="relations">
<h3>Related Topics</h3>
<ul>
  <li><a href="index.html">Documentation overview</a><ul>
  <li><a href="usage.html">Usage</a><ul>
      <li>Previous: <a href="alignment.html" title="previous chapter">Alignment</a></li>
      <li>Next: <a href="retain.html" title="next chapter">Retain</a></li>
  </ul></li>
  </ul></li>
</ul>
</div>
<search id="searchbox" style="display: none" role="search">
  <h3 id="searchlabel">Quick search</h3>
    <div class="searchformwrapper">
    <form class="search" action="search.html" method="get">
      <input type="text" name="q" aria-labelledby="searchlabel" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false"/>
      <input type="submit" value="Go" />
    </form>
    </div>
</search>
<script>document.getElementById('searchbox').style.display = "block"</script>








        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="footer">
      &#169;2024, Pengchuan Sun.
      
      |
      Powered by <a href="https://www.sphinx-doc.org/">Sphinx 8.1.3</a>
      &amp; <a href="https://alabaster.readthedocs.io">Alabaster 0.7.16</a>
      
      |
      <a href="_sources/alignment_tree.rst.txt"
          rel="nofollow">Page source</a>
    </div>

    

    
  </body>
</html>