<!DOCTYPE html>

<html lang="en" data-content_root="./">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>BlockInfo &#8212; WGDI 0.71 documentation</title>
    <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=d1102ebc" />
    <link rel="stylesheet" type="text/css" href="_static/alabaster.css?v=12dfc556" />
    <script src="_static/documentation_options.js?v=89bdf08e"></script>
    <script src="_static/doctools.js?v=9bcbadda"></script>
    <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Correspondence" href="correspondence.html" />
    <link rel="prev" title="Non-synonymous (Ka) and synonymous (Ks)" href="ks.html" />
   
  <link rel="stylesheet" href="_static/custom.css" type="text/css" />
  

  
  

  </head><body>
  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          

          <div class="body" role="main">
            
  <section id="blockinfo">
<h1>BlockInfo<a class="headerlink" href="#blockinfo" title="Link to this heading">Â¶</a></h1>
<p>The color distinction in the dot plot, the collinearity result and the Ks result are integrated into one file.
This file contains the main information to achieve the purpose of easily screening the collinearity fragments.</p>
<p class="rubric">Parameters</p>
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Parameters</p></td>
<td><p>Standards and instructions</p></td>
</tr>
<tr class="row-even"><td><p>collinearity</p></td>
<td><p>Type: <strong>file</strong> |    Default: <strong>-</strong></p>
<p>Colinearscan or MCScanX result file or gene pairs divided by tab.</p>
</td>
</tr>
<tr class="row-odd"><td><p>score</p></td>
<td><p>Type: <strong>int</strong>  |   Default: <strong>100</strong></p>
<p>Score value in the blast results.</p>
</td>
</tr>
<tr class="row-even"><td><p>evalue</p></td>
<td><p>Type: <strong>float</strong>   |  Default: <strong>1e-5</strong></p>
<p>Evalue value in blast result.</p>
</td>
</tr>
<tr class="row-odd"><td><p>repeat_number</p></td>
<td><p>Type: <strong>int</strong>  |   Default: <strong>10</strong></p>
<p>The maximum number of homologous genes is allowed to remove more than part of the population.</p>
</td>
</tr>
<tr class="row-even"><td><p>position</p></td>
<td><p>Type: {<strong>order</strong>}  |   Default: <strong>order</strong></p>
<p>The position of a gene corresponds to the gff file.</p>
</td>
</tr>
<tr class="row-odd"><td><p>ks</p></td>
<td><p>Type: <strong>file</strong>  |    Default: <strong>-</strong></p>
<p>Ks calculation results.</p>
</td>
</tr>
<tr class="row-even"><td><p>ks_col</p></td>
<td><p>Type: <strong>str</strong>    Default: <strong>NG86</strong></p>
<p>The result calculated by ks_YN00 or other methods is a single column.</p>
</td>
</tr>
<tr class="row-odd"><td><p>savefile</p></td>
<td><p>Type: <strong>file</strong>   |  Default: * <strong>.csv</strong></p>
<p>Result file.</p>
</td>
</tr>
</tbody>
</table>
<p>Use command to enter the folder <code class="docutils literal notranslate"><span class="pre">wgdi</span> <span class="pre">-bi</span> <span class="pre">?</span> <span class="pre">&gt;&gt;</span> <span class="pre">total.conf</span></code> Take out the parameter file.</p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="p">[</span><span class="n">blockinfo</span><span class="p">]</span>
<span class="n">blast</span> <span class="o">=</span> <span class="n">blast</span> <span class="n">file</span>
<span class="n">gff1</span> <span class="o">=</span>  <span class="n">gff1</span> <span class="n">file</span>
<span class="n">gff2</span> <span class="o">=</span>  <span class="n">gff2</span> <span class="n">file</span>
<span class="n">lens1</span> <span class="o">=</span> <span class="n">lens1</span> <span class="n">file</span>
<span class="n">lens2</span> <span class="o">=</span> <span class="n">lens2</span> <span class="n">file</span>
<span class="n">collinearity</span> <span class="o">=</span> <span class="n">collinearity</span> <span class="n">file</span>
<span class="n">score</span> <span class="o">=</span> <span class="mi">100</span>
<span class="n">evalue</span> <span class="o">=</span> <span class="mf">1e-5</span>
<span class="n">repeat_number</span> <span class="o">=</span> <span class="mi">10</span>
<span class="n">position</span> <span class="o">=</span> <span class="n">order</span>
<span class="n">ks</span> <span class="o">=</span> <span class="n">ks</span> <span class="n">file</span>
<span class="n">ks_col</span> <span class="o">=</span> <span class="n">ks_NG86</span>
<span class="n">savefile</span> <span class="o">=</span> <span class="n">block</span> <span class="n">information</span> <span class="p">(</span><span class="o">*.</span><span class="n">csv</span><span class="p">)</span>
</pre></div>
</div>
<p class="rubric">Quick start</p>
<p>After the parameters are modified properly, then run <code class="docutils literal notranslate"><span class="pre">wgdi</span> <span class="pre">-bi</span> <span class="pre">total.conf</span></code></p>
<p class="rubric">Example</p>
<p>The original results are easily accessible at <a class="reference external" href="https://github.com/SunPengChuan/wgdi-example">wgdi-example</a></p>
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>columns</p></td>
<td><p>Information</p></td>
</tr>
<tr class="row-even"><td><p>id</p></td>
<td><p>Type: <strong>str</strong></p>
<p>Unique id.</p>
</td>
</tr>
<tr class="row-odd"><td><p>chr1,chr2</p></td>
<td><p>Type: <strong>int</strong></p>
<p>Two chromosomes corresponding to collinearity block.</p>
</td>
</tr>
<tr class="row-even"><td><p>start1,end1</p></td>
<td><p>Type: <strong>int</strong></p>
<p>The region of chromosome 1 on collinearity block.</p>
</td>
</tr>
<tr class="row-odd"><td><p>start2,end2</p></td>
<td><p>Type: <strong>int</strong></p>
<p>The region of chromosome 2 on collinearity block.</p>
</td>
</tr>
<tr class="row-even"><td><p>pvalue</p></td>
<td><p>Type: <strong>float</strong></p>
<p>Evaluate the compactness and uniqueness of collinear blocks, the range is 0-1, the better collinearity range is 0-0.2.</p>
</td>
</tr>
<tr class="row-odd"><td><p>length</p></td>
<td><p>Type: <strong>int</strong></p>
<p>The length of collinearity block.</p>
</td>
</tr>
<tr class="row-even"><td><p>ks_median</p></td>
<td><p>Type: <strong>float</strong></p>
<p>The median of ks value on collinearity block.</p>
</td>
</tr>
<tr class="row-odd"><td><p>ks_average</p></td>
<td><p>Type: <strong>float</strong></p>
<p>The average of ks value on collinearity block.</p>
</td>
</tr>
<tr class="row-even"><td><p>homo1-5</p></td>
<td><p>Type: <strong>float</strong></p>
<p>The average of scores of different color dots (red=1, blue=0, gray=-1) on synteny blocks.</p>
</td>
</tr>
<tr class="row-odd"><td><p>block1,block2</p></td>
<td><p>Type: <strong>str</strong></p>
<p>Gene pairs in collinear blocks</p>
</td>
</tr>
<tr class="row-even"><td><p>ks</p></td>
<td><p>Type: <strong>str</strong></p>
<p>Output result of parameter <cite>ks</cite>.</p>
</td>
</tr>
<tr class="row-odd"><td><p>tandem_ratio</p></td>
<td><p>Type: <strong>float</strong></p>
<p>The density of tandem repeats, in general, a synteny block greater than 0.5 is unreliable.</p>
</td>
</tr>
<tr class="row-even"><td><p>density1,density2</p></td>
<td><p>Type: <strong>float</strong></p>
<p>Density of collinear blocks.</p>
</td>
</tr>
<tr class="row-odd"><td><p>class1,class2</p></td>
<td><p>Type: <strong>str</strong></p>
<p>Class of collinear blocks.</p>
</td>
</tr>
</tbody>
</table>
</section>


          </div>
          
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="Main">
        <div class="sphinxsidebarwrapper">
<h1 class="logo"><a href="index.html">WGDI</a></h1>








<h3>Navigation</h3>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="Introduction.html">Introduction</a></li>
<li class="toctree-l1"><a class="reference internal" href="Installation.html">Installation</a></li>
<li class="toctree-l1 current"><a class="reference internal" href="usage.html">Usage</a><ul class="current">
<li class="toctree-l2"><a class="reference internal" href="dotplot.html">Dotplot</a></li>
<li class="toctree-l2"><a class="reference internal" href="collinearity.html">Improved collinearity</a></li>
<li class="toctree-l2"><a class="reference internal" href="ks.html">Non-synonymous (Ka) and synonymous (Ks)</a></li>
<li class="toctree-l2 current"><a class="current reference internal" href="#">BlockInfo</a></li>
<li class="toctree-l2"><a class="reference internal" href="correspondence.html">Correspondence</a></li>
<li class="toctree-l2"><a class="reference internal" href="blockks.html">BlockKs</a></li>
<li class="toctree-l2"><a class="reference internal" href="kspeaks.html">KsPeaks</a></li>
<li class="toctree-l2"><a class="reference internal" href="peaksfit.html">PeaksFit</a></li>
<li class="toctree-l2"><a class="reference internal" href="ksfigure.html">KsFigure</a></li>
<li class="toctree-l2"><a class="reference internal" href="polyploidy_classification.html">Polyploid classification</a></li>
<li class="toctree-l2"><a class="reference internal" href="ancestral_karyotype.html">ancestral_karyotype</a></li>
<li class="toctree-l2"><a class="reference internal" href="ancestral_karyotype_repertoire.html">ancestral_karyotype_repertoire</a></li>
<li class="toctree-l2"><a class="reference internal" href="karyotype_mapping.html">karyotype_mapping</a></li>
<li class="toctree-l2"><a class="reference internal" href="karyotype.html">Karyotype</a></li>
<li class="toctree-l2"><a class="reference internal" href="shared_fusions.html">Shared_fusions</a></li>
<li class="toctree-l2"><a class="reference internal" href="fusion_positions_database.html">Fusion_positions_database</a></li>
<li class="toctree-l2"><a class="reference internal" href="fusion_detetion.html">Fusions_detection</a></li>
<li class="toctree-l2"><a class="reference internal" href="alignment.html">Alignment</a></li>
<li class="toctree-l2"><a class="reference internal" href="alignment_tree.html">Alignmenttrees</a></li>
<li class="toctree-l2"><a class="reference internal" href="retain.html">Retain</a></li>
<li class="toctree-l2"><a class="reference internal" href="pindex.html">P-index</a></li>
<li class="toctree-l2"><a class="reference internal" href="circos.html">Circos</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="Examples.html">Examples</a></li>
<li class="toctree-l1"><a class="reference internal" href="Changelog.html">Changelog</a></li>
</ul>

<div class="relations">
<h3>Related Topics</h3>
<ul>
  <li><a href="index.html">Documentation overview</a><ul>
  <li><a href="usage.html">Usage</a><ul>
      <li>Previous: <a href="ks.html" title="previous chapter">Non-synonymous (Ka) and synonymous (Ks)</a></li>
      <li>Next: <a href="correspondence.html" title="next chapter">Correspondence</a></li>
  </ul></li>
  </ul></li>
</ul>
</div>
<search id="searchbox" style="display: none" role="search">
  <h3 id="searchlabel">Quick search</h3>
    <div class="searchformwrapper">
    <form class="search" action="search.html" method="get">
      <input type="text" name="q" aria-labelledby="searchlabel" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false"/>
      <input type="submit" value="Go" />
    </form>
    </div>
</search>
<script>document.getElementById('searchbox').style.display = "block"</script>








        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="footer">
      &#169;2024, Pengchuan Sun.
      
      |
      Powered by <a href="https://www.sphinx-doc.org/">Sphinx 8.1.3</a>
      &amp; <a href="https://alabaster.readthedocs.io">Alabaster 0.7.16</a>
      
      |
      <a href="_sources/blockinfo.rst.txt"
          rel="nofollow">Page source</a>
    </div>

    

    
  </body>
</html>