<!DOCTYPE html>
<html class="writer-html5" lang="en" data-content_root="./">
<head>
  <meta charset="utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>Changelog &mdash; WGDI latest documentation</title>
      <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=80d5e7a1" />
      <link rel="stylesheet" type="text/css" href="_static/css/theme.css?v=19f00094" />

  
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
        <script src="_static/jquery.js?v=5d32c60e"></script>
        <script src="_static/_sphinx_javascript_frameworks_compat.js?v=2cd50e6c"></script>
        <script src="_static/documentation_options.js?v=c6e86fd7"></script>
        <script src="_static/doctools.js?v=9bcbadda"></script>
        <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="_static/js/theme.js"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="prev" title="Examples" href="Examples.html" /> 
</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >

          
          
          <a href="index.html" class="icon icon-home">
            WGDI
          </a>
              <div class="version">
                0.7.1
              </div>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <ul class="current">
<li class="toctree-l1"><a class="reference internal" href="Introduction.html">Introduction</a></li>
<li class="toctree-l1"><a class="reference internal" href="Installation.html">Installation</a></li>
<li class="toctree-l1"><a class="reference internal" href="Usage.html">Usage</a></li>
<li class="toctree-l1"><a class="reference internal" href="Examples.html">Examples</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Changelog</a></li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">WGDI</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="index.html" class="icon icon-home" aria-label="Home"></a></li>
      <li class="breadcrumb-item active">Changelog</li>
      <li class="wy-breadcrumbs-aside">
            <a href="_sources/Changelog.rst.txt" rel="nofollow"> View page source</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section id="changelog">
<h1>Changelog<a class="headerlink" href="#changelog" title="Link to this heading"></a></h1>
<p class="rubric">0.7.1</p>
<ul class="simple">
<li><p>Added extract the fusion positions dataset (-fpd).</p></li>
<li><p>Added determine whether these fusion events occur in other genomes (-fd).</p></li>
<li><p>Improved the karyotype_mapping (-km) effect.</p></li>
<li><p>Fixed the problem caused by the Python version, now it is compatible with version 3.12.</p></li>
</ul>
<p class="rubric">0.6.5</p>
<ul class="simple">
<li><p>Fixed some issues (-sf).</p></li>
<li><p>Added new tips to avoid some errors.</p></li>
</ul>
<p class="rubric">0.6.4</p>
<ul class="simple">
<li><p>Fixed the problem caused by the Python version, now it is compatible with version 3.11.3.</p></li>
</ul>
<p class="rubric">0.6.3</p>
<ul class="simple">
<li><p>Fixed some issues (-ks, -sf).</p></li>
</ul>
<p class="rubric">0.6.2</p>
<ul class="simple">
<li><p>Added find shared fusions between species (-sf).</p></li>
</ul>
<p class="rubric">0.6.1</p>
<ul class="simple">
<li><p>Fixed issue with alignment (-a). Only version 0.6.0 has this bug.</p></li>
</ul>
<p class="rubric">0.6.0</p>
<ul class="simple">
<li><p>Fixed issue with improved collinearity (-icl).</p></li>
<li><p>Added a parameter ‘tandem_ratio’ to blockinfo (-bi).</p></li>
</ul>
<p class="rubric">0.5.9</p>
<ul class="simple">
<li><p>Update the improved collinearity (-icl). Faster than before, but lower than MCscanX, JCVI.</p></li>
<li><p>Fixed issue with ancestral karyotype repertoire (-akr).</p></li>
</ul>
<p class="rubric">0.5.8</p>
<ul class="simple">
<li><p>Fixed issue with gene names (-ks).</p></li>
</ul>
<p class="rubric">0.5.7</p>
<ul class="simple">
<li><p>Fixed issue with chromosome order (-ak).</p></li>
<li><p>Fixed issue with gene names (-ks).  This version is not fixed, please install the latest version.</p></li>
</ul>
<p class="rubric">0.5.5 and 0.5.6</p>
<ul class="simple">
<li><p>Add ancestral karyotype (-ak)</p></li>
<li><p>Add ancestral karyotype repertoire (-akr)</p></li>
</ul>
<p class="rubric">0.5.4</p>
<ul class="simple">
<li><p>Improved the alignmenttrees (-km) effect.</p></li>
<li><p>little change (-at).</p></li>
</ul>
<p class="rubric">0.5.3</p>
<ul class="simple">
<li><p>Fixed legend issue with (-kf).</p></li>
<li><p>Fixed calculate Ks issue with (-ks).</p></li>
<li><p>Improved the karyotype_mapping (-km) effect.</p></li>
<li><p>Improved the alignmenttrees (-at) effect.</p></li>
</ul>
<p class="rubric">0.5.2</p>
<ul class="simple">
<li><p>Fixed some bugs.</p></li>
</ul>
<p class="rubric">0.5.1</p>
<ul class="simple">
<li><p>Fixed the error of the command (-conf).</p></li>
<li><p>Improved the karyotype_mapping (-km) effect.</p></li>
<li><p>Added the available data set of alignmenttree (-at). Low copy data set (for example, single-copy_groups.tsv of sonicparanoid2 software).</p></li>
</ul>
<p class="rubric">0.4.9</p>
<ul class="simple">
<li><p>The latest version adds karyotype_mapping (-km) and karyotype (-k) display.</p></li>
<li><p>The latest version changes the calculation of extracting pvalue from collinearity (-icl), making this parameter more sensitive. Therefore, it is recommended to set to 0.2 instead of 0.05.</p></li>
<li><p>The latest version has also changed the drawing display of ksfigure (-kf) to make it more beautiful.</p></li>
</ul>
</section>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="Examples.html" class="btn btn-neutral float-left" title="Examples" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright 2024, PengChuan Sun.</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>