<!DOCTYPE html>

<html lang="en" data-content_root="./">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>Improved collinearity &#8212; WGDI 0.71 documentation</title>
    <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=d1102ebc" />
    <link rel="stylesheet" type="text/css" href="_static/alabaster.css?v=12dfc556" />
    <script src="_static/documentation_options.js?v=89bdf08e"></script>
    <script src="_static/doctools.js?v=9bcbadda"></script>
    <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Non-synonymous (Ka) and synonymous (Ks)" href="ks.html" />
    <link rel="prev" title="Dotplot" href="dotplot.html" />
   
  <link rel="stylesheet" href="_static/custom.css" type="text/css" />
  

  
  

  </head><body>
  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          

          <div class="body" role="main">
            
  <section id="improved-collinearity">
<h1>Improved collinearity<a class="headerlink" href="#improved-collinearity" title="Link to this heading">¶</a></h1>
<p>The algorithm for extracting collinearity is based on the dynamic programming algorithm, similar to the ColinearScan and MCScan.</p>
<p class="rubric">Parameters</p>
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Parameters</p></td>
<td><p>Standards and instructions</p></td>
</tr>
<tr class="row-even"><td><p>multiple</p></td>
<td><p>Type: <strong>int</strong>   |    Default: <strong>1</strong></p>
<p>The best number of homologous genes shown with red dots.</p>
</td>
</tr>
<tr class="row-odd"><td><p>evalue</p></td>
<td><p>Type: <strong>float</strong>  |  Default: <strong>1e-5</strong></p>
<p>Evalue in the blast result.</p>
</td>
</tr>
<tr class="row-even"><td><p>score</p></td>
<td><p>Type: <strong>int</strong>  |  Default: <strong>100</strong></p>
<p>Score value in the blast results.</p>
</td>
</tr>
<tr class="row-odd"><td><p>grading</p></td>
<td><p>Type: <strong>int</strong> , <strong>int</strong> , <strong>int</strong>  |  Default: <strong>50</strong>, <strong>40</strong> , <strong>25</strong></p>
<p>Assign different scores based on the colors in the dotplot, with a default of 50 for red, 40 for blue, and 25 for gray.</p>
</td>
</tr>
<tr class="row-even"><td><p>mg</p></td>
<td><p>Type: <strong>int</strong> , <strong>int</strong>   |  Default: <strong>40</strong>, <strong>40</strong></p>
<p>The maximum gap(mg) value is an important parameter for detecting collinear regions.</p>
</td>
</tr>
<tr class="row-odd"><td><p>comparison</p></td>
<td><p>Type: { <strong>genomes</strong>, <strong>chromosomes</strong> }  |   Default: <strong>genomes</strong></p>
<p>When comparing with the fusion positions database, use ‘chromosomes’; otherwise, use ‘genomes’.</p>
</td>
</tr>
<tr class="row-even"><td><p>pvalue</p></td>
<td><p>Type: <strong>float</strong>  |    Default: <strong>1</strong></p>
<p>Evaluate the compactness and uniqueness of collinear blocks, the range is 0-1, and the better collinearity range is 0-0.2.</p>
</td>
</tr>
<tr class="row-odd"><td><p>repeat_number</p></td>
<td><p>Type: <strong>int</strong> |   Default: <strong>10</strong></p>
<p>The maximum number of homologous genes is allowed to remove more than part of the population.</p>
</td>
</tr>
<tr class="row-even"><td><p>process</p></td>
<td><p>Type: <strong>int</strong>  |  Default: <strong>8</strong></p>
<p>Number of multiple processes.</p>
</td>
</tr>
<tr class="row-odd"><td><p>position</p></td>
<td><p>Type: <strong>order</strong>  |  Default: <strong>order</strong></p>
<p>The position of a gene corresponds to the gff file.</p>
</td>
</tr>
</tbody>
</table>
<p>Use command to enter the folder <code class="docutils literal notranslate"><span class="pre">wgdi</span> <span class="pre">-icl</span> <span class="pre">?</span> <span class="pre">&gt;&gt;</span> <span class="pre">total.conf</span></code> Take out the parameter file.</p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="p">[</span><span class="n">collinearity</span><span class="p">]</span>
<span class="n">gff1</span> <span class="o">=</span> <span class="n">gff1</span> <span class="n">file</span>
<span class="n">gff2</span> <span class="o">=</span> <span class="n">gff2</span> <span class="n">file</span>
<span class="n">lens1</span> <span class="o">=</span> <span class="n">lens1</span> <span class="n">file</span>
<span class="n">lens2</span> <span class="o">=</span> <span class="n">lens2</span> <span class="n">file</span>
<span class="n">blast</span> <span class="o">=</span> <span class="n">blast</span> <span class="n">file</span>
<span class="n">blast_reverse</span> <span class="o">=</span> <span class="n">false</span>
<span class="n">comparison</span> <span class="o">=</span> <span class="n">genomes</span>
<span class="n">multiple</span>  <span class="o">=</span> <span class="mi">1</span>
<span class="n">process</span> <span class="o">=</span> <span class="mi">8</span>
<span class="n">evalue</span> <span class="o">=</span> <span class="mf">1e-5</span>
<span class="n">score</span> <span class="o">=</span> <span class="mi">100</span>
<span class="n">grading</span> <span class="o">=</span> <span class="mi">50</span><span class="p">,</span><span class="mi">30</span><span class="p">,</span><span class="mi">25</span>
<span class="n">mg</span> <span class="o">=</span> <span class="mi">25</span><span class="p">,</span><span class="mi">25</span>
<span class="n">pvalue</span> <span class="o">=</span> <span class="mi">1</span>
<span class="n">repeat_number</span> <span class="o">=</span> <span class="mi">20</span>
<span class="n">positon</span> <span class="o">=</span> <span class="n">order</span>
<span class="n">savefile</span> <span class="o">=</span> <span class="n">collinearity</span> <span class="n">file</span>
</pre></div>
</div>
<p class="rubric">Quick start</p>
<p>After the parameters are modified properly, then run <code class="docutils literal notranslate"><span class="pre">wgdi</span> <span class="pre">-icl</span> <span class="pre">total.conf</span></code></p>
<p class="rubric">Example</p>
<p>The original results are easily accessible at <a class="reference external" href="https://github.com/SunPengChuan/wgdi-example">wgdi-example</a></p>
<img alt="_images/collinearity.png" class="align-left" src="_images/collinearity.png" />
</section>


          </div>
          
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="Main">
        <div class="sphinxsidebarwrapper">
<h1 class="logo"><a href="index.html">WGDI</a></h1>








<h3>Navigation</h3>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="Introduction.html">Introduction</a></li>
<li class="toctree-l1"><a class="reference internal" href="Installation.html">Installation</a></li>
<li class="toctree-l1 current"><a class="reference internal" href="usage.html">Usage</a><ul class="current">
<li class="toctree-l2"><a class="reference internal" href="dotplot.html">Dotplot</a></li>
<li class="toctree-l2 current"><a class="current reference internal" href="#">Improved collinearity</a></li>
<li class="toctree-l2"><a class="reference internal" href="ks.html">Non-synonymous (Ka) and synonymous (Ks)</a></li>
<li class="toctree-l2"><a class="reference internal" href="blockinfo.html">BlockInfo</a></li>
<li class="toctree-l2"><a class="reference internal" href="correspondence.html">Correspondence</a></li>
<li class="toctree-l2"><a class="reference internal" href="blockks.html">BlockKs</a></li>
<li class="toctree-l2"><a class="reference internal" href="kspeaks.html">KsPeaks</a></li>
<li class="toctree-l2"><a class="reference internal" href="peaksfit.html">PeaksFit</a></li>
<li class="toctree-l2"><a class="reference internal" href="ksfigure.html">KsFigure</a></li>
<li class="toctree-l2"><a class="reference internal" href="polyploidy_classification.html">Polyploid classification</a></li>
<li class="toctree-l2"><a class="reference internal" href="ancestral_karyotype.html">ancestral_karyotype</a></li>
<li class="toctree-l2"><a class="reference internal" href="ancestral_karyotype_repertoire.html">ancestral_karyotype_repertoire</a></li>
<li class="toctree-l2"><a class="reference internal" href="karyotype_mapping.html">karyotype_mapping</a></li>
<li class="toctree-l2"><a class="reference internal" href="karyotype.html">Karyotype</a></li>
<li class="toctree-l2"><a class="reference internal" href="shared_fusions.html">Shared_fusions</a></li>
<li class="toctree-l2"><a class="reference internal" href="fusion_positions_database.html">Fusion_positions_database</a></li>
<li class="toctree-l2"><a class="reference internal" href="fusion_detetion.html">Fusions_detection</a></li>
<li class="toctree-l2"><a class="reference internal" href="alignment.html">Alignment</a></li>
<li class="toctree-l2"><a class="reference internal" href="alignment_tree.html">Alignmenttrees</a></li>
<li class="toctree-l2"><a class="reference internal" href="retain.html">Retain</a></li>
<li class="toctree-l2"><a class="reference internal" href="pindex.html">P-index</a></li>
<li class="toctree-l2"><a class="reference internal" href="circos.html">Circos</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="Examples.html">Examples</a></li>
<li class="toctree-l1"><a class="reference internal" href="Changelog.html">Changelog</a></li>
</ul>

<div class="relations">
<h3>Related Topics</h3>
<ul>
  <li><a href="index.html">Documentation overview</a><ul>
  <li><a href="usage.html">Usage</a><ul>
      <li>Previous: <a href="dotplot.html" title="previous chapter">Dotplot</a></li>
      <li>Next: <a href="ks.html" title="next chapter">Non-synonymous (Ka) and synonymous (Ks)</a></li>
  </ul></li>
  </ul></li>
</ul>
</div>
<search id="searchbox" style="display: none" role="search">
  <h3 id="searchlabel">Quick search</h3>
    <div class="searchformwrapper">
    <form class="search" action="search.html" method="get">
      <input type="text" name="q" aria-labelledby="searchlabel" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false"/>
      <input type="submit" value="Go" />
    </form>
    </div>
</search>
<script>document.getElementById('searchbox').style.display = "block"</script>








        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="footer">
      &#169;2024, Pengchuan Sun.
      
      |
      Powered by <a href="https://www.sphinx-doc.org/">Sphinx 8.1.3</a>
      &amp; <a href="https://alabaster.readthedocs.io">Alabaster 0.7.16</a>
      
      |
      <a href="_sources/collinearity.rst.txt"
          rel="nofollow">Page source</a>
    </div>

    

    
  </body>
</html>