<!DOCTYPE html>

<html lang="en" data-content_root="./">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>Introduction &#8212; WGDI 0.71 documentation</title>
    <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=d1102ebc" />
    <link rel="stylesheet" type="text/css" href="_static/alabaster.css?v=12dfc556" />
    <script src="_static/documentation_options.js?v=89bdf08e"></script>
    <script src="_static/doctools.js?v=9bcbadda"></script>
    <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Installation" href="Installation.html" />
    <link rel="prev" title="Welcome to WGDI’s Documentation!" href="index.html" />
   
  <link rel="stylesheet" href="_static/custom.css" type="text/css" />
  

  
  

  </head><body>
  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          

          <div class="body" role="main">
            
  <section id="introduction">
<h1>Introduction<a class="headerlink" href="#introduction" title="Link to this heading">¶</a></h1>
<img alt="_images/workflow.png" src="_images/workflow.png" />
<p>The WGDI workﬂow consists of three main parts: <strong>(1)</strong> Polyploidy inference using Dotplot, Collinearity extraction, Ks distributions
<strong>(2)</strong> Hierarchical inference of genomic homology resulted from recursive paleopolyploidizations.
<strong>(3)</strong> Subgenomic and ancestral genome reconstruction and other evolutionary scenarios.</p>
<p>WGDI contains multiple subroutines, the user only needs to modify the configuration file simply, and then enter the name of the subroutine to be executed,
Such as <code class="docutils literal notranslate"><span class="pre">wgdi</span> <span class="pre">-d</span> <span class="pre">your.conf</span></code>.  In the following, we will describe in detail the subroutines of WGDI software.</p>
<p><strong>WGDI subroutine and function</strong></p>
<table class="docutils align-default">
<thead>
<tr class="row-odd"><th class="head" colspan="2"><p>Parameters</p></th>
<th class="head"><p>Functions</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>-h</p></td>
<td><p>Help</p></td>
<td><p>Show help message and exit</p></td>
</tr>
<tr class="row-odd"><td><p>-v</p></td>
<td><p>Version</p></td>
<td><p>Show program’s version number</p></td>
</tr>
<tr class="row-even"><td><p>-d</p></td>
<td><p>DotPlot</p></td>
<td><p>Show homologous gene dotplot</p></td>
</tr>
<tr class="row-odd"><td><p>-icl</p></td>
<td><p>Collinearity</p></td>
<td><p>Improved version of ColinearScan</p></td>
</tr>
<tr class="row-even"><td><p>-ks</p></td>
<td><p>CalKs</p></td>
<td><p>Calculate Ka/Ks for homologous gene pairs by YN00</p></td>
</tr>
<tr class="row-odd"><td><p>-bi</p></td>
<td><p>BlockInfo</p></td>
<td><p>Collinearity and Ks speculate whole genome duplication</p></td>
</tr>
<tr class="row-even"><td><p>-c</p></td>
<td><p>Correspondence</p></td>
<td><p>Extract event-related genomic alignment</p></td>
</tr>
<tr class="row-odd"><td><p>-bk</p></td>
<td><p>BlockKs</p></td>
<td><p>Show Ks of blocks in a dotplot</p></td>
</tr>
<tr class="row-even"><td><p>-kp</p></td>
<td><p>KsPeaks</p></td>
<td><p>A simple way to get ks peaks</p></td>
</tr>
<tr class="row-odd"><td><p>-pf</p></td>
<td><p>PeaksFit</p></td>
<td><p>Gaussian fitting of ks distribution</p></td>
</tr>
<tr class="row-even"><td><p>-pc</p></td>
<td><p>polyploidy_classification</p></td>
<td><p>Show event-related genomic alignment in a dotplot</p></td>
</tr>
<tr class="row-odd"><td><p>-km</p></td>
<td><p>karotype_mapping</p></td>
<td><p>Mapping from the known karyotype result to this species</p></td>
</tr>
<tr class="row-even"><td><p>-k</p></td>
<td><p>karotype</p></td>
<td><p>Show genome evolution from reconstructed ancestors</p></td>
</tr>
<tr class="row-odd"><td><p>-sf</p></td>
<td><p>Shared_fusion</p></td>
<td><p>Quickly find shared fusions between species</p></td>
</tr>
<tr class="row-even"><td><p>-fpd</p></td>
<td><p>Fusion_positions_database</p></td>
<td><p>Extract the fusion positions dataset</p></td>
</tr>
<tr class="row-odd"><td><p>-fd</p></td>
<td><p>Fusion_detection</p></td>
<td><p>Determine whether these fusion events occur in other genomes</p></td>
</tr>
<tr class="row-even"><td><p>-a</p></td>
<td><p>Alignment</p></td>
<td><p>Alignment of hierarchical and event-related gene collinearity</p></td>
</tr>
<tr class="row-odd"><td><p>-at</p></td>
<td><p>AlignmentTrees</p></td>
<td><p>Phylogenetic trees constructed by collinear genes</p></td>
</tr>
<tr class="row-even"><td><p>-p</p></td>
<td><p>P-index</p></td>
<td><p>Polyploidy-index characterize the degree of divergence among subgenomes of a polyploidy</p></td>
</tr>
<tr class="row-odd"><td><p>-r</p></td>
<td><p>Retain</p></td>
<td><p>Show subgenomes in gene retention or genome fractionation</p></td>
</tr>
<tr class="row-even"><td><p>-ci</p></td>
<td><p>Circos</p></td>
<td><p>A simple way to run circos</p></td>
</tr>
<tr class="row-odd"><td><p>-conf</p></td>
<td><p>Configure</p></td>
<td><p>Display and modify the environment variable</p></td>
</tr>
</tbody>
</table>
</section>


          </div>
          
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="Main">
        <div class="sphinxsidebarwrapper">
<h1 class="logo"><a href="index.html">WGDI</a></h1>








<h3>Navigation</h3>
<ul class="current">
<li class="toctree-l1 current"><a class="current reference internal" href="#">Introduction</a></li>
<li class="toctree-l1"><a class="reference internal" href="Installation.html">Installation</a></li>
<li class="toctree-l1"><a class="reference internal" href="usage.html">Usage</a></li>
<li class="toctree-l1"><a class="reference internal" href="Examples.html">Examples</a></li>
<li class="toctree-l1"><a class="reference internal" href="Changelog.html">Changelog</a></li>
</ul>

<div class="relations">
<h3>Related Topics</h3>
<ul>
  <li><a href="index.html">Documentation overview</a><ul>
      <li>Previous: <a href="index.html" title="previous chapter">Welcome to WGDI’s Documentation!</a></li>
      <li>Next: <a href="Installation.html" title="next chapter">Installation</a></li>
  </ul></li>
</ul>
</div>
<search id="searchbox" style="display: none" role="search">
  <h3 id="searchlabel">Quick search</h3>
    <div class="searchformwrapper">
    <form class="search" action="search.html" method="get">
      <input type="text" name="q" aria-labelledby="searchlabel" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false"/>
      <input type="submit" value="Go" />
    </form>
    </div>
</search>
<script>document.getElementById('searchbox').style.display = "block"</script>








        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="footer">
      &#169;2024, Pengchuan Sun.
      
      |
      Powered by <a href="https://www.sphinx-doc.org/">Sphinx 8.1.3</a>
      &amp; <a href="https://alabaster.readthedocs.io">Alabaster 0.7.16</a>
      
      |
      <a href="_sources/Introduction.rst.txt"
          rel="nofollow">Page source</a>
    </div>

    

    
  </body>
</html>