<!DOCTYPE html>
<html class="writer-html5" lang="En" data-content_root="./">
<head>
  <meta charset="utf-8" /><meta name="viewport" content="width=device-width, initial-scale=1" />

  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  <title>karyotype_mapping &mdash; WGDI latest documentation</title>
      <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=80d5e7a1" />
      <link rel="stylesheet" type="text/css" href="_static/css/theme.css?v=19f00094" />

  
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
        <script src="_static/jquery.js?v=5d32c60e"></script>
        <script src="_static/_sphinx_javascript_frameworks_compat.js?v=2cd50e6c"></script>
        <script src="_static/documentation_options.js?v=2ab17cd4"></script>
        <script src="_static/doctools.js?v=888ff710"></script>
        <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <script src="_static/js/theme.js"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Karyotype" href="karyotype.html" />
    <link rel="prev" title="ancestral_karyotype_repertoire" href="ancestral_karyotype_repertoire.html" /> 
</head>

<body class="wy-body-for-nav"> 
  <div class="wy-grid-for-nav">
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >

          
          
          <a href="index.html" class="icon icon-home">
            WGDI
          </a>
              <div class="version">
                0.6.5
              </div>
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" aria-label="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>
        </div><div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="Navigation menu">
              <ul class="current">
<li class="toctree-l1"><a class="reference internal" href="Introduction.html">Introduction</a></li>
<li class="toctree-l1"><a class="reference internal" href="Installation.html">Installation</a></li>
<li class="toctree-l1 current"><a class="reference internal" href="usage.html">Usage</a><ul class="current">
<li class="toctree-l2"><a class="reference internal" href="dotplot.html">Dotplot</a></li>
<li class="toctree-l2"><a class="reference internal" href="collinearity.html">Improved collinearity</a></li>
<li class="toctree-l2"><a class="reference internal" href="ks.html">Non-synonymous (Ka) and synonymous (Ks)</a></li>
<li class="toctree-l2"><a class="reference internal" href="blockinfo.html">BlockInfo</a></li>
<li class="toctree-l2"><a class="reference internal" href="correspondence.html">Correspondence</a></li>
<li class="toctree-l2"><a class="reference internal" href="blockks.html">BlockKs</a></li>
<li class="toctree-l2"><a class="reference internal" href="kspeaks.html">KsPeaks</a></li>
<li class="toctree-l2"><a class="reference internal" href="peaksfit.html">PeaksFit</a></li>
<li class="toctree-l2"><a class="reference internal" href="ksfigure.html">KsFigure</a></li>
<li class="toctree-l2"><a class="reference internal" href="polyploidy_classification.html">Polyploid classification</a></li>
<li class="toctree-l2"><a class="reference internal" href="ancestral_karyotype.html">ancestral_karyotype</a></li>
<li class="toctree-l2"><a class="reference internal" href="ancestral_karyotype_repertoire.html">ancestral_karyotype_repertoire</a></li>
<li class="toctree-l2 current"><a class="current reference internal" href="#">karyotype_mapping</a></li>
<li class="toctree-l2"><a class="reference internal" href="karyotype.html">Karyotype</a></li>
<li class="toctree-l2"><a class="reference internal" href="shared_fusions.html">Shared_fusions</a></li>
<li class="toctree-l2"><a class="reference internal" href="alignment.html">Alignment</a></li>
<li class="toctree-l2"><a class="reference internal" href="alignment_tree.html">Alignmenttrees</a></li>
<li class="toctree-l2"><a class="reference internal" href="retain.html">Retain</a></li>
<li class="toctree-l2"><a class="reference internal" href="pindex.html">P-index</a></li>
<li class="toctree-l2"><a class="reference internal" href="circos.html">Circos</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="examples.html">Examples</a></li>
<li class="toctree-l1"><a class="reference internal" href="changelog.html">Changelog</a></li>
</ul>

        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap"><nav class="wy-nav-top" aria-label="Mobile navigation menu" >
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">WGDI</a>
      </nav>

      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="Page navigation">
  <ul class="wy-breadcrumbs">
      <li><a href="index.html" class="icon icon-home" aria-label="Home"></a></li>
          <li class="breadcrumb-item"><a href="usage.html">Usage</a></li>
      <li class="breadcrumb-item active">karyotype_mapping</li>
      <li class="wy-breadcrumbs-aside">
            <a href="_sources/karyotype_mapping.rst.txt" rel="nofollow"> View page source</a>
      </li>
  </ul>
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
             
  <section id="karyotype-mapping">
<h1>karyotype_mapping<a class="headerlink" href="#karyotype-mapping" title="Link to this heading"></a></h1>
<p>Mapping from the known karyotype result to this species.</p>
<p class="rubric">Parameters</p>
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Parameters</p></td>
<td><p>Standards and instructions</p></td>
</tr>
<tr class="row-even"><td><p>blast_reverse</p></td>
<td><p>Type: <strong>bool</strong>   |    Default: <strong>false</strong></p>
<p>The first two columns of the blast result swap positions.</p>
</td>
</tr>
<tr class="row-odd"><td><p>score</p></td>
<td><p>Type: <strong>int</strong>    |     Default: <strong>100</strong></p>
<p>Score value in blast result.</p>
</td>
</tr>
<tr class="row-even"><td><p>evalue</p></td>
<td><p>Type: <strong>float</strong>  |  Default: <strong>1e-5</strong></p>
<p>Evalue value in blast result.</p>
</td>
</tr>
<tr class="row-odd"><td><p>repeat_number</p></td>
<td><p>Type: <strong>int</strong>  |  Default: <strong>10</strong></p>
<p>The maximum number of homologous genes is allowed to remove more than part of the population.</p>
</td>
</tr>
<tr class="row-even"><td><p>ancestor_left</p></td>
<td><p>Type: <strong>file</strong>  |  Default: <strong>none</strong></p>
<p>Ancestor location file (Only one of (‘left’, ‘top’) can be reserved in this conf file, and the other should be deleted)</p>
</td>
</tr>
<tr class="row-odd"><td><p>ancestor_top</p></td>
<td><p>Type: <strong>file</strong>  |  Default: <strong>none</strong></p>
<p>Ancestor location file (Only one of (‘left’, ‘top’) can be reserved in this conf file, and the other should be deleted)</p>
</td>
</tr>
<tr class="row-even"><td><p>blockinfo</p></td>
<td><p>Type: <strong>file</strong>  |    Default: <strong>-</strong></p>
<p>blockinfo.csv filtered according to certain conditions ‘-c’</p>
</td>
</tr>
<tr class="row-odd"><td><p>the_other_lens</p></td>
<td><p>Type: <strong>file</strong>  |    Default: <strong>-</strong></p>
<p>The lens file for the species for which you want to generate ancestor files.</p>
</td>
</tr>
<tr class="row-even"><td><p>limit_length</p></td>
<td><p>Type: <strong>int</strong>  |    Default: <strong>5</strong></p>
<p>Show the minimum length of blocks.</p>
</td>
</tr>
<tr class="row-odd"><td><p>the_other_ancestor_file</p></td>
<td><p>Type: <strong>file</strong>   |  Default: * <strong>.csv</strong></p>
<p>The resulting file.</p>
</td>
</tr>
</tbody>
</table>
<p>Use command to enter the folder <code class="docutils literal notranslate"><span class="pre">wgdi</span> <span class="pre">-km</span> <span class="pre">?</span> <span class="pre">&gt;&gt;</span> <span class="pre">total.conf</span></code> Take out the parameter file.</p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="p">[</span><span class="n">karyotype_mapping</span><span class="p">]</span>
<span class="n">blast</span> <span class="o">=</span> <span class="n">blast</span> <span class="n">file</span>
<span class="n">blast_reverse</span> <span class="o">=</span> <span class="n">false</span>
<span class="n">gff1</span> <span class="o">=</span> <span class="n">gff1</span> <span class="n">file</span>
<span class="n">gff2</span> <span class="o">=</span> <span class="n">gff2</span> <span class="n">file</span>
<span class="n">score</span> <span class="o">=</span> <span class="mi">100</span>
<span class="n">evalue</span> <span class="o">=</span> <span class="mf">1e-5</span>
<span class="n">repeat_number</span> <span class="o">=</span> <span class="mi">5</span>
<span class="n">ancestor_left</span> <span class="o">=</span> <span class="n">ancestor</span> <span class="n">location</span> <span class="n">file</span> <span class="p">(</span><span class="n">Only</span> <span class="n">one</span> <span class="n">of</span> <span class="p">(</span><span class="s1">&#39;left&#39;</span><span class="p">,</span> <span class="s1">&#39;top&#39;</span><span class="p">)</span> <span class="n">can</span> <span class="n">be</span> <span class="n">reserved</span><span class="p">)</span>
<span class="n">ancestor_top</span> <span class="o">=</span> <span class="n">ancestor</span> <span class="n">location</span> <span class="n">file</span>
<span class="n">the_other_lens</span> <span class="o">=</span> <span class="n">the</span> <span class="n">other</span> <span class="n">lens</span> <span class="n">file</span>
<span class="n">blockinfo</span> <span class="o">=</span> <span class="n">block</span> <span class="n">information</span> <span class="p">(</span><span class="o">*.</span><span class="n">csv</span><span class="p">)</span>
<span class="n">blockinfo_reverse</span> <span class="o">=</span> <span class="n">false</span>
<span class="n">limit_length</span> <span class="o">=</span> <span class="mi">5</span>
<span class="n">the_other_ancestor_file</span> <span class="o">=</span>  <span class="n">result</span> <span class="n">file</span>
</pre></div>
</div>
<p class="rubric">Quick start</p>
<p>The original results are easily accessible at <a class="reference external" href="https://github.com/SunPengChuan/wgdi-example">wgdi-example</a></p>
</section>


           </div>
          </div>
          <footer><div class="rst-footer-buttons" role="navigation" aria-label="Footer">
        <a href="ancestral_karyotype_repertoire.html" class="btn btn-neutral float-left" title="ancestral_karyotype_repertoire" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
        <a href="karyotype.html" class="btn btn-neutral float-right" title="Karyotype" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>&#169; Copyright 2021, PengChuan Sun.</p>
  </div>

  Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    provided by <a href="https://readthedocs.org">Read the Docs</a>.
   

</footer>
        </div>
      </div>
    </section>
  </div>
  <script>
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script> 

</body>
</html>