<!DOCTYPE html>

<html lang="en" data-content_root="./">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>P-index &#8212; WGDI 0.71 documentation</title>
    <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=d1102ebc" />
    <link rel="stylesheet" type="text/css" href="_static/alabaster.css?v=12dfc556" />
    <script src="_static/documentation_options.js?v=89bdf08e"></script>
    <script src="_static/doctools.js?v=9bcbadda"></script>
    <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Circos" href="circos.html" />
    <link rel="prev" title="Retain" href="retain.html" />
   
  <link rel="stylesheet" href="_static/custom.css" type="text/css" />
  

  
  

  </head><body>
  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          

          <div class="body" role="main">
            
  <section id="p-index">
<h1>P-index<a class="headerlink" href="#p-index" title="Link to this heading">¶</a></h1>
<p>The polyploidy index(P-index) is used to characterize the degree of divergence between subgenomes of a polyploidy, to find whether there has been a balanced or unbalanced gene removal from the homoeologous regions.</p>
<p class="rubric">Parameters</p>
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Parameters</p></td>
<td><p>Standards and instructions</p></td>
</tr>
<tr class="row-even"><td><p>alignment</p></td>
<td><p>Type:<strong>file</strong>  |      Default: <strong>-</strong></p>
<p>Output result of parameter <cite>a</cite></p>
</td>
</tr>
<tr class="row-odd"><td><p>gap</p></td>
<td><p>type: <strong>int</strong>  |   Default: <strong>50</strong></p>
<p>The size of the sliding window.</p>
</td>
</tr>
<tr class="row-even"><td><p>colors</p></td>
<td><p>Type: { <strong>color1,color2,color3,–</strong> }    Default: <strong>red,blue,green</strong></p>
<p>Set multiple sets of colors based on grouping, split with a comma.</p>
</td>
</tr>
<tr class="row-odd"><td><p>retention</p></td>
<td><p>Type: <strong>float</strong>  |    Default: <strong>0.05</strong></p>
<p>The region where the retention rate of the sub-genome relative to the reference genome is low, which is 0.05 by default.</p>
</td>
</tr>
<tr class="row-even"><td><p>diff</p></td>
<td><p>Type: <strong>float</strong> |  Default: <strong>0.05</strong></p>
<p>More than this value is considered a significant difference.</p>
</td>
</tr>
<tr class="row-odd"><td><p>remove_delta</p></td>
<td><p>Type: <strong>bool</strong>   | Default: <strong>true</strong></p>
<p>Whether to remove the parameter retention.</p>
</td>
</tr>
<tr class="row-even"><td><p>savefile</p></td>
<td><p>Type:<strong>file</strong>  |      Default: <strong>-</strong></p>
<p>Results of the drawing.</p>
</td>
</tr>
</tbody>
</table>
<p class="rubric">Parameters</p>
<p>Use command to enter the folder <code class="docutils literal notranslate"><span class="pre">wgdi</span> <span class="pre">-p</span> <span class="pre">?</span> <span class="pre">&gt;&gt;</span> <span class="pre">total.conf</span></code> Take out the parameter file.</p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="p">[</span><span class="n">pindex</span><span class="p">]</span>
<span class="n">alignment</span> <span class="o">=</span> <span class="n">alignment</span> <span class="n">file</span>
<span class="n">gff</span> <span class="o">=</span> <span class="n">gff</span> <span class="n">file</span>
<span class="n">lens</span> <span class="o">=</span><span class="n">lens</span> <span class="n">file</span>
<span class="n">gap</span> <span class="o">=</span> <span class="mi">50</span>
<span class="n">retention</span> <span class="o">=</span> <span class="mf">0.05</span>
<span class="n">diff</span> <span class="o">=</span> <span class="mf">0.05</span>
<span class="n">remove_delta</span> <span class="o">=</span> <span class="p">(</span><span class="n">true</span><span class="o">/</span><span class="n">false</span><span class="p">)</span>
<span class="n">savefile</span> <span class="o">=</span> <span class="n">result</span> <span class="n">file</span>
</pre></div>
</div>
<p class="rubric">Quick start</p>
<p>After the parameters are modified properly, then run <code class="docutils literal notranslate"><span class="pre">wgdi</span> <span class="pre">-p</span> <span class="pre">total.conf</span></code></p>
<p class="rubric">Example</p>
<p>The detailed explanation is in the published  <a class="reference external" href="https://doi.org/10.3389/fgene.2019.00807">article</a></p>
</section>


          </div>
          
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="Main">
        <div class="sphinxsidebarwrapper">
<h1 class="logo"><a href="index.html">WGDI</a></h1>








<h3>Navigation</h3>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="Introduction.html">Introduction</a></li>
<li class="toctree-l1"><a class="reference internal" href="Installation.html">Installation</a></li>
<li class="toctree-l1 current"><a class="reference internal" href="usage.html">Usage</a><ul class="current">
<li class="toctree-l2"><a class="reference internal" href="dotplot.html">Dotplot</a></li>
<li class="toctree-l2"><a class="reference internal" href="collinearity.html">Improved collinearity</a></li>
<li class="toctree-l2"><a class="reference internal" href="ks.html">Non-synonymous (Ka) and synonymous (Ks)</a></li>
<li class="toctree-l2"><a class="reference internal" href="blockinfo.html">BlockInfo</a></li>
<li class="toctree-l2"><a class="reference internal" href="correspondence.html">Correspondence</a></li>
<li class="toctree-l2"><a class="reference internal" href="blockks.html">BlockKs</a></li>
<li class="toctree-l2"><a class="reference internal" href="kspeaks.html">KsPeaks</a></li>
<li class="toctree-l2"><a class="reference internal" href="peaksfit.html">PeaksFit</a></li>
<li class="toctree-l2"><a class="reference internal" href="ksfigure.html">KsFigure</a></li>
<li class="toctree-l2"><a class="reference internal" href="polyploidy_classification.html">Polyploid classification</a></li>
<li class="toctree-l2"><a class="reference internal" href="ancestral_karyotype.html">ancestral_karyotype</a></li>
<li class="toctree-l2"><a class="reference internal" href="ancestral_karyotype_repertoire.html">ancestral_karyotype_repertoire</a></li>
<li class="toctree-l2"><a class="reference internal" href="karyotype_mapping.html">karyotype_mapping</a></li>
<li class="toctree-l2"><a class="reference internal" href="karyotype.html">Karyotype</a></li>
<li class="toctree-l2"><a class="reference internal" href="shared_fusions.html">Shared_fusions</a></li>
<li class="toctree-l2"><a class="reference internal" href="fusion_positions_database.html">Fusion_positions_database</a></li>
<li class="toctree-l2"><a class="reference internal" href="fusion_detetion.html">Fusions_detection</a></li>
<li class="toctree-l2"><a class="reference internal" href="alignment.html">Alignment</a></li>
<li class="toctree-l2"><a class="reference internal" href="alignment_tree.html">Alignmenttrees</a></li>
<li class="toctree-l2"><a class="reference internal" href="retain.html">Retain</a></li>
<li class="toctree-l2 current"><a class="current reference internal" href="#">P-index</a></li>
<li class="toctree-l2"><a class="reference internal" href="circos.html">Circos</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="Examples.html">Examples</a></li>
<li class="toctree-l1"><a class="reference internal" href="Changelog.html">Changelog</a></li>
</ul>

<div class="relations">
<h3>Related Topics</h3>
<ul>
  <li><a href="index.html">Documentation overview</a><ul>
  <li><a href="usage.html">Usage</a><ul>
      <li>Previous: <a href="retain.html" title="previous chapter">Retain</a></li>
      <li>Next: <a href="circos.html" title="next chapter">Circos</a></li>
  </ul></li>
  </ul></li>
</ul>
</div>
<search id="searchbox" style="display: none" role="search">
  <h3 id="searchlabel">Quick search</h3>
    <div class="searchformwrapper">
    <form class="search" action="search.html" method="get">
      <input type="text" name="q" aria-labelledby="searchlabel" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false"/>
      <input type="submit" value="Go" />
    </form>
    </div>
</search>
<script>document.getElementById('searchbox').style.display = "block"</script>








        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="footer">
      &#169;2024, Pengchuan Sun.
      
      |
      Powered by <a href="https://www.sphinx-doc.org/">Sphinx 8.1.3</a>
      &amp; <a href="https://alabaster.readthedocs.io">Alabaster 0.7.16</a>
      
      |
      <a href="_sources/pindex.rst.txt"
          rel="nofollow">Page source</a>
    </div>

    

    
  </body>
</html>