<!DOCTYPE html>

<html lang="en" data-content_root="./">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>Polyploid classification &#8212; WGDI 0.71 documentation</title>
    <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=d1102ebc" />
    <link rel="stylesheet" type="text/css" href="_static/alabaster.css?v=12dfc556" />
    <script src="_static/documentation_options.js?v=89bdf08e"></script>
    <script src="_static/doctools.js?v=9bcbadda"></script>
    <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Ancestral_karyotype" href="ancestral_karyotype.html" />
    <link rel="prev" title="KsFigure" href="ksfigure.html" />
   
  <link rel="stylesheet" href="_static/custom.css" type="text/css" />
  

  
  

  </head><body>
  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          

          <div class="body" role="main">
            
  <section id="polyploid-classification">
<h1>Polyploid classification<a class="headerlink" href="#polyploid-classification" title="Link to this heading">Â¶</a></h1>
<p>Polyploid distinguish among subgenomes.</p>
<p class="rubric">Parameters</p>
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Parameters</p></td>
<td><p>Standards and instructions</p></td>
</tr>
<tr class="row-even"><td><p>blockinfo</p></td>
<td><p>Type: <strong>file</strong>  |    Default: <strong>-</strong></p>
<p>Output result of parameter <cite>bi</cite></p>
</td>
</tr>
<tr class="row-odd"><td><p>ancestor_left</p></td>
<td><p>Type: <strong>file</strong>  |  Default: <strong>none</strong></p>
<p>The ancestral chromosome region of the species on the left of dotplot.</p>
</td>
</tr>
<tr class="row-even"><td><p>ancestor_top</p></td>
<td><p>Type: <strong>file</strong>  |  Default: <strong>none</strong></p>
<p>The ancestral chromosome region of the species on the top of dotplot.</p>
</td>
</tr>
<tr class="row-odd"><td><p>diff</p></td>
<td><p>Type: <strong>float</strong> |  Default: <strong>0.05</strong></p>
<p>More than this value is considered a significant difference.</p>
</td>
</tr>
<tr class="row-even"><td><p>classid</p></td>
<td><p>Type: <strong>str</strong>  |    Default: <strong>class1,class2</strong></p>
<p>The class name of the grouping according to the divided ancestral area.</p>
</td>
</tr>
<tr class="row-odd"><td><p>same_protochromosome</p></td>
<td><p>Type: <strong>bool</strong>  |    Default: <strong>False</strong></p>
<p>A synteny block spans two protochromosomes of the same color.</p>
</td>
</tr>
<tr class="row-even"><td><p>savefile</p></td>
<td><p>Type: <strong>file</strong>   |  Default: * <strong>.csv</strong></p>
<p>The resulting file.</p>
</td>
</tr>
</tbody>
</table>
<p>Use command to enter the folder <code class="docutils literal notranslate"><span class="pre">wgdi</span> <span class="pre">-pc</span> <span class="pre">?</span> <span class="pre">&gt;&gt;</span> <span class="pre">total.conf</span></code> Take out the parameter file.</p>
<div class="highlight-python notranslate"><div class="highlight"><pre><span></span><span class="p">[</span><span class="n">polyploidy</span> <span class="n">classification</span><span class="p">]</span>
<span class="n">blockinfo</span> <span class="o">=</span> <span class="n">block</span> <span class="n">information</span> <span class="p">(</span><span class="o">*.</span><span class="n">csv</span><span class="p">)</span>
<span class="n">ancestor_left</span> <span class="o">=</span> <span class="n">file</span>
<span class="n">ancestor_top</span> <span class="o">=</span> <span class="n">file</span>
<span class="n">diff</span> <span class="o">=</span> <span class="mf">0.05</span>
<span class="n">classid</span> <span class="o">=</span> <span class="n">class1</span><span class="p">,</span><span class="n">class2</span>
<span class="n">same_protochromosome</span> <span class="o">=</span>  <span class="kc">False</span>
<span class="n">savefile</span> <span class="o">=</span> <span class="n">result</span> <span class="n">file</span><span class="p">(</span><span class="o">.</span><span class="n">csv</span><span class="p">)</span>
</pre></div>
</div>
<p class="rubric">Quick start</p>
<p>The original results are easily accessible at <a class="reference external" href="https://github.com/SunPengChuan/wgdi-example">wgdi-example</a></p>
</section>


          </div>
          
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="Main">
        <div class="sphinxsidebarwrapper">
<h1 class="logo"><a href="index.html">WGDI</a></h1>








<h3>Navigation</h3>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="introduction.html">Introduction</a></li>
<li class="toctree-l1"><a class="reference internal" href="installation.html">Installation</a></li>
<li class="toctree-l1 current"><a class="reference internal" href="usage.html">Usage</a><ul class="current">
<li class="toctree-l2"><a class="reference internal" href="dotplot.html">Dotplot</a></li>
<li class="toctree-l2"><a class="reference internal" href="collinearity.html">Improved collinearity</a></li>
<li class="toctree-l2"><a class="reference internal" href="ks.html">Non-synonymous (Ka) and synonymous (Ks)</a></li>
<li class="toctree-l2"><a class="reference internal" href="blockinfo.html">BlockInfo</a></li>
<li class="toctree-l2"><a class="reference internal" href="correspondence.html">Correspondence</a></li>
<li class="toctree-l2"><a class="reference internal" href="blockks.html">BlockKs</a></li>
<li class="toctree-l2"><a class="reference internal" href="kspeaks.html">KsPeaks</a></li>
<li class="toctree-l2"><a class="reference internal" href="peaksfit.html">PeaksFit</a></li>
<li class="toctree-l2"><a class="reference internal" href="ksfigure.html">KsFigure</a></li>
<li class="toctree-l2 current"><a class="current reference internal" href="#">Polyploid classification</a></li>
<li class="toctree-l2"><a class="reference internal" href="ancestral_karyotype.html">Ancestral_karyotype</a></li>
<li class="toctree-l2"><a class="reference internal" href="ancestral_karyotype_repertoire.html">Ancestral_karyotype_repertoire</a></li>
<li class="toctree-l2"><a class="reference internal" href="karyotype_mapping.html">karyotype_mapping</a></li>
<li class="toctree-l2"><a class="reference internal" href="karyotype.html">Karyotype</a></li>
<li class="toctree-l2"><a class="reference internal" href="shared_fusions.html">Shared_fusions</a></li>
<li class="toctree-l2"><a class="reference internal" href="fusion_positions_database.html">Fusion_positions_database</a></li>
<li class="toctree-l2"><a class="reference internal" href="fusion_detetion.html">Fusions_detection</a></li>
<li class="toctree-l2"><a class="reference internal" href="alignment.html">Alignment</a></li>
<li class="toctree-l2"><a class="reference internal" href="alignment_tree.html">Alignmenttrees</a></li>
<li class="toctree-l2"><a class="reference internal" href="retain.html">Retain</a></li>
<li class="toctree-l2"><a class="reference internal" href="pindex.html">P-index</a></li>
<li class="toctree-l2"><a class="reference internal" href="circos.html">Circos</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="examples.html">Examples</a></li>
<li class="toctree-l1"><a class="reference internal" href="changelog.html">Changelog</a></li>
</ul>

<div class="relations">
<h3>Related Topics</h3>
<ul>
  <li><a href="index.html">Documentation overview</a><ul>
  <li><a href="usage.html">Usage</a><ul>
      <li>Previous: <a href="ksfigure.html" title="previous chapter">KsFigure</a></li>
      <li>Next: <a href="ancestral_karyotype.html" title="next chapter">Ancestral_karyotype</a></li>
  </ul></li>
  </ul></li>
</ul>
</div>
<search id="searchbox" style="display: none" role="search">
  <h3 id="searchlabel">Quick search</h3>
    <div class="searchformwrapper">
    <form class="search" action="search.html" method="get">
      <input type="text" name="q" aria-labelledby="searchlabel" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false"/>
      <input type="submit" value="Go" />
    </form>
    </div>
</search>
<script>document.getElementById('searchbox').style.display = "block"</script>








        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="footer">
      &#169;2024, Pengchuan Sun.
      
      |
      Powered by <a href="https://www.sphinx-doc.org/">Sphinx 8.1.3</a>
      &amp; <a href="https://alabaster.readthedocs.io">Alabaster 0.7.16</a>
      
      |
      <a href="_sources/polyploidy_classification.rst.txt"
          rel="nofollow">Page source</a>
    </div>

    

    
  </body>
</html>