<!DOCTYPE html>

<html lang="en" data-content_root="./">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>Usage &#8212; WGDI 0.71 documentation</title>
    <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=d1102ebc" />
    <link rel="stylesheet" type="text/css" href="_static/alabaster.css?v=12dfc556" />
    <script src="_static/documentation_options.js?v=89bdf08e"></script>
    <script src="_static/doctools.js?v=9bcbadda"></script>
    <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Dotplot" href="dotplot.html" />
    <link rel="prev" title="Installation" href="Installation.html" />
   
  <link rel="stylesheet" href="_static/custom.css" type="text/css" />
  

  
  

  </head><body>
  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          

          <div class="body" role="main">
            
  <section id="usage">
<h1>Usage<a class="headerlink" href="#usage" title="Link to this heading">¶</a></h1>
<p>We support the use of <strong>WGDI</strong> to complete the work on the icon number.</p>
<div class="toctree-wrapper compound">
<ul>
<li class="toctree-l1"><a class="reference internal" href="dotplot.html">Dotplot</a></li>
<li class="toctree-l1"><a class="reference internal" href="collinearity.html">Improved collinearity</a></li>
<li class="toctree-l1"><a class="reference internal" href="ks.html">Non-synonymous (Ka) and synonymous (Ks)</a></li>
<li class="toctree-l1"><a class="reference internal" href="blockinfo.html">BlockInfo</a></li>
<li class="toctree-l1"><a class="reference internal" href="correspondence.html">Correspondence</a></li>
<li class="toctree-l1"><a class="reference internal" href="blockks.html">BlockKs</a></li>
<li class="toctree-l1"><a class="reference internal" href="kspeaks.html">KsPeaks</a></li>
<li class="toctree-l1"><a class="reference internal" href="peaksfit.html">PeaksFit</a></li>
<li class="toctree-l1"><a class="reference internal" href="ksfigure.html">KsFigure</a></li>
<li class="toctree-l1"><a class="reference internal" href="polyploidy_classification.html">Polyploid classification</a></li>
<li class="toctree-l1"><a class="reference internal" href="ancestral_karyotype.html">ancestral_karyotype</a></li>
<li class="toctree-l1"><a class="reference internal" href="ancestral_karyotype_repertoire.html">ancestral_karyotype_repertoire</a></li>
<li class="toctree-l1"><a class="reference internal" href="karyotype_mapping.html">karyotype_mapping</a></li>
<li class="toctree-l1"><a class="reference internal" href="karyotype.html">Karyotype</a></li>
<li class="toctree-l1"><a class="reference internal" href="shared_fusions.html">Shared_fusions</a></li>
<li class="toctree-l1"><a class="reference internal" href="fusion_positions_database.html">Fusion_positions_database</a></li>
<li class="toctree-l1"><a class="reference internal" href="fusion_detetion.html">Fusions_detection</a></li>
<li class="toctree-l1"><a class="reference internal" href="alignment.html">Alignment</a></li>
<li class="toctree-l1"><a class="reference internal" href="alignment_tree.html">Alignmenttrees</a></li>
<li class="toctree-l1"><a class="reference internal" href="retain.html">Retain</a></li>
<li class="toctree-l1"><a class="reference internal" href="pindex.html">P-index</a></li>
<li class="toctree-l1"><a class="reference internal" href="circos.html">Circos</a></li>
</ul>
</div>
<p><strong>Common file</strong></p>
<ul class="simple">
<li><p>conf</p></li>
</ul>
<p>The conf file contains parameters required for the corresponding operation, which are read when WGDI is performed.
Using <code class="docutils literal notranslate"><span class="pre">wgdi</span> <span class="pre">-*</span> <span class="pre">?</span> <span class="pre">&gt;</span> <span class="pre">*.conf</span></code> to get the configuration file in the current directory and modify it to run.
If you don’t know which files are needed, you can view it through <code class="docutils literal notranslate"><span class="pre">wgdi</span> <span class="pre">-*</span> <span class="pre">?</span></code> or <code class="docutils literal notranslate"><span class="pre">wgdi</span> <span class="pre">-*</span> <span class="pre">help</span></code>  or <code class="docutils literal notranslate"><span class="pre">wgdi</span> <span class="pre">-*</span> <span class="pre">example</span></code>. These three commands are equivalent.</p>
<p>In conf file: <strong>gff1</strong> , <strong>lens1</strong> , <strong>genome1_name</strong> and <strong>gff2</strong>,  <strong>lens2</strong>, <strong>genome2_name</strong> represent the files of species 1 and 2 respectively.
We will no longer explain these in this documentation.</p>
<ul class="simple">
<li><p>gff</p></li>
</ul>
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Column</p></td>
<td><p>Information</p></td>
<td><p>Explanation</p></td>
</tr>
<tr class="row-even"><td><p>1</p></td>
<td><p>Chr</p></td>
<td><p>Chromosome number</p></td>
</tr>
<tr class="row-odd"><td><p>2</p></td>
<td><p>Id</p></td>
<td><p>Gene name</p></td>
</tr>
<tr class="row-even"><td><p>3</p></td>
<td><p>Strat</p></td>
<td><p>The starting location of a gene</p></td>
</tr>
<tr class="row-odd"><td><p>4</p></td>
<td><p>End</p></td>
<td><p>The ending location of a  gene</p></td>
</tr>
<tr class="row-even"><td><p>5</p></td>
<td><p>Direction</p></td>
<td><p>Direction of a gene sequence</p></td>
</tr>
<tr class="row-odd"><td><p>6</p></td>
<td><p>Order</p></td>
<td><p>Order of each chromosome, starting from 1</p></td>
</tr>
<tr class="row-even"><td><p>7</p></td>
<td><p>Original</p></td>
<td><p>Original id and  not read</p></td>
</tr>
</tbody>
</table>
<ul class="simple">
<li><p>lens</p></li>
</ul>
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Column</p></td>
<td><p>Information</p></td>
<td><p>Explanation</p></td>
</tr>
<tr class="row-even"><td><p>1</p></td>
<td><p>Chr</p></td>
<td><p>Chromosome number</p></td>
</tr>
<tr class="row-odd"><td><p>2</p></td>
<td><p>Length</p></td>
<td><p>Length of chromosome sequences</p></td>
</tr>
<tr class="row-even"><td><p>3</p></td>
<td><p>Number</p></td>
<td><p>Number of chromosome genes</p></td>
</tr>
</tbody>
</table>
<ul class="simple">
<li><p>blast</p></li>
</ul>
<p>The protein-coding genes from each genome were compared against itself and other genomes using <a class="reference external" href="ftp://ftp.ncbi.nlm.nih.gov/blast/executables/LATEST/">BLASTP</a>  (e-value &lt; 10-5, and outfmt = 6) or other similar protein sequence searching software ( <a class="reference external" href="https://github.com/soedinglab/MMseqs2">MMseqs2</a> , <a class="reference external" href="https://github.com/bbuchfink/diamond">DIAMOND</a> ).</p>
<ul class="simple">
<li><p>ancestor file</p></li>
</ul>
<p>Required documents for karyotype evolution analysis</p>
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Column</p></td>
<td><p>Information</p></td>
<td><p>Explanation</p></td>
</tr>
<tr class="row-even"><td><p>1</p></td>
<td><p>Chr</p></td>
<td><p>Chromosome number</p></td>
</tr>
<tr class="row-odd"><td><p>2</p></td>
<td><p>Start</p></td>
<td><p>Homologous regions of these protochromosomes in this genome</p></td>
</tr>
<tr class="row-even"><td><p>3</p></td>
<td><p>End</p></td>
<td><p>Homologous regions of these protochromosomes in this genome</p></td>
</tr>
<tr class="row-odd"><td><p>4</p></td>
<td><p>Color</p></td>
<td><p>These protochromosomes with different colors</p></td>
</tr>
<tr class="row-even"><td><p>5</p></td>
<td><p>Subgenomes</p></td>
<td><p>Subgenomes according to protochromosomes</p></td>
</tr>
</tbody>
</table>
<ul class="simple">
<li><p>Fusino postions file</p></li>
</ul>
<p>Required documents for shared fusion events</p>
<table class="docutils align-default">
<tbody>
<tr class="row-odd"><td><p>Column</p></td>
<td><p>Information</p></td>
<td><p>Explanation</p></td>
</tr>
<tr class="row-even"><td><p>1</p></td>
<td><p>Chr</p></td>
<td><p>Chromosome number</p></td>
</tr>
<tr class="row-odd"><td><p>2</p></td>
<td><p>Breakpoint</p></td>
<td><p>The breakpoints of fusions, with multiple breakpoints listed on separate lines.</p></td>
</tr>
<tr class="row-even"><td><p>3</p></td>
<td><p>Mark</p></td>
<td><p>The unique marks of unique fusions</p></td>
</tr>
</tbody>
</table>
<p><strong>Tips</strong></p>
<ul class="simple">
<li><p>You can use <code class="docutils literal notranslate"><span class="pre">wgdi</span> <span class="pre">-conf</span> <span class="pre">?</span> <span class="pre">&gt;</span> <span class="pre">total.conf</span></code> generates a <strong>total.conf</strong> file with all parameters, and when you modify the parameters and run WGDI, WGDI will only read the parameters corresponding to the total.conf file to execute your command.</p></li>
<li><p>When a folder runs WGDI, WGDI automatically generates results for you in the background, and you can exit the folder and go to the next folder to start working.</p></li>
<li><p>WGDI performs the <strong>conf</strong> file for the current folder, so you can copy the <strong>conf</strong> file in bulk and make parameter modifications appied to the target folder.</p></li>
</ul>
</section>


          </div>
          
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="Main">
        <div class="sphinxsidebarwrapper">
<h1 class="logo"><a href="index.html">WGDI</a></h1>








<h3>Navigation</h3>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="Introduction.html">Introduction</a></li>
<li class="toctree-l1"><a class="reference internal" href="Installation.html">Installation</a></li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Usage</a><ul>
<li class="toctree-l2"><a class="reference internal" href="dotplot.html">Dotplot</a></li>
<li class="toctree-l2"><a class="reference internal" href="collinearity.html">Improved collinearity</a></li>
<li class="toctree-l2"><a class="reference internal" href="ks.html">Non-synonymous (Ka) and synonymous (Ks)</a></li>
<li class="toctree-l2"><a class="reference internal" href="blockinfo.html">BlockInfo</a></li>
<li class="toctree-l2"><a class="reference internal" href="correspondence.html">Correspondence</a></li>
<li class="toctree-l2"><a class="reference internal" href="blockks.html">BlockKs</a></li>
<li class="toctree-l2"><a class="reference internal" href="kspeaks.html">KsPeaks</a></li>
<li class="toctree-l2"><a class="reference internal" href="peaksfit.html">PeaksFit</a></li>
<li class="toctree-l2"><a class="reference internal" href="ksfigure.html">KsFigure</a></li>
<li class="toctree-l2"><a class="reference internal" href="polyploidy_classification.html">Polyploid classification</a></li>
<li class="toctree-l2"><a class="reference internal" href="ancestral_karyotype.html">ancestral_karyotype</a></li>
<li class="toctree-l2"><a class="reference internal" href="ancestral_karyotype_repertoire.html">ancestral_karyotype_repertoire</a></li>
<li class="toctree-l2"><a class="reference internal" href="karyotype_mapping.html">karyotype_mapping</a></li>
<li class="toctree-l2"><a class="reference internal" href="karyotype.html">Karyotype</a></li>
<li class="toctree-l2"><a class="reference internal" href="shared_fusions.html">Shared_fusions</a></li>
<li class="toctree-l2"><a class="reference internal" href="fusion_positions_database.html">Fusion_positions_database</a></li>
<li class="toctree-l2"><a class="reference internal" href="fusion_detetion.html">Fusions_detection</a></li>
<li class="toctree-l2"><a class="reference internal" href="alignment.html">Alignment</a></li>
<li class="toctree-l2"><a class="reference internal" href="alignment_tree.html">Alignmenttrees</a></li>
<li class="toctree-l2"><a class="reference internal" href="retain.html">Retain</a></li>
<li class="toctree-l2"><a class="reference internal" href="pindex.html">P-index</a></li>
<li class="toctree-l2"><a class="reference internal" href="circos.html">Circos</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="Examples.html">Examples</a></li>
<li class="toctree-l1"><a class="reference internal" href="Changelog.html">Changelog</a></li>
</ul>

<div class="relations">
<h3>Related Topics</h3>
<ul>
  <li><a href="index.html">Documentation overview</a><ul>
      <li>Previous: <a href="Installation.html" title="previous chapter">Installation</a></li>
      <li>Next: <a href="dotplot.html" title="next chapter">Dotplot</a></li>
  </ul></li>
</ul>
</div>
<search id="searchbox" style="display: none" role="search">
  <h3 id="searchlabel">Quick search</h3>
    <div class="searchformwrapper">
    <form class="search" action="search.html" method="get">
      <input type="text" name="q" aria-labelledby="searchlabel" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false"/>
      <input type="submit" value="Go" />
    </form>
    </div>
</search>
<script>document.getElementById('searchbox').style.display = "block"</script>








        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="footer">
      &#169;2024, Pengchuan Sun.
      
      |
      Powered by <a href="https://www.sphinx-doc.org/">Sphinx 8.1.3</a>
      &amp; <a href="https://alabaster.readthedocs.io">Alabaster 0.7.16</a>
      
      |
      <a href="_sources/Usage.rst.txt"
          rel="nofollow">Page source</a>
    </div>

    

    
  </body>
</html>